intake_speed: 1
outtake_speed: 1
outtake_stop_delay: 0.5